frappe/core/doctype/submission_queue/submission_queue.json

```diff
@@ -88,24 +88,14 @@
  ],
  "index_web_pages_for_search": 1,
  "links": [],
- "modified": "2023-01-23 12:45:53.997708",
+ "modified": "2023-08-28 18:48:56.242178",
  "modified_by": "Administrator",
  "module": "Core",
  "name": "Submission Queue",
  "naming_rule": "Random",
  "owner": "Administrator",
  "permissions": [
   {
-   "email": 1,
-   "export": 1,
-   "print": 1,
-   "read": 1,
-   "report": 1,
-   "role": "System Manager",
-   "share": 1
-  },
-  {
-   "if_owner": 1,
    "read": 1,
    "role": "All"
```

Comment:
is there a role which is something along like "all the system users can access the docs" - iirc "all" is applicable for website users/guests as well?